testing-guidance/workflow-tests/scripting/script-canvas-tests.md

```diff
@@ -60,54 +64,72 @@ Responsive Script Canvas editor with all 5 basic tools enabled. No Asset Process
+|--------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Creating a user function and calling its input/output from another graph                   | [![](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/images/SCeditor_basic_userFunction00.png)](https://github.com/o3de/sig-content/blob/main/testing-guidance/workflow-tests/scripting/images/SCeditor_basic_userFunction00.png) <ol><li>Use Script Canvas editor to create a graph file with enter/exit nodelings.</li><li>Place complex logic or entity behavior between the enter/exit nodelings.</li><li>Give the graph file a unique recognizable name and save it.</li><li>Call the user function from another graph by finding it in the node palette or right click context menu [![Calling a user function from another graph](https://github.com/o3de/sig-content/raw/main/testing-guidance/workflow-tests/scripting/images/SCeditor_basic_userFunction01.png)](https://github.com/o3de/sig-content/blob/main/testing-guidance/workflow-tests/scripting/images/SCeditor_basic_userFunction01.png)</li></ol> **Example Functions:** <ol><li>Two color parameters and returns a blended color.</li><li>Entity parameter and returns the number of child entities.</li><li>No parameters or output but triggers behavior. | <ul><li>Recursive functions are **NOT** supported at the moment</li><li>Modifications to the user function file should require scripts calling the function to also be modified.</li> |
```

Comment:
The image reference at the beginning doesn't have any context. Restructure or add text to give context to this image.